pkg/helm/helm_helpers.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helm
+
+import (
+	"regexp"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+
+	"github.com/kanisterio/kanister/pkg/field"
+	"github.com/kanisterio/kanister/pkg/log"
+)
+
+type k8sObj struct {
+	ObjKind  string            `json:"kind"`
+	MetaData metav1.ObjectMeta `json:"metadata"`
+}
+
+type K8sObjectType string
+
+type Component struct {
+	k8sType K8sObjectType
+	name    string
+}
+
+/*
+	This file contains utility functions primarily to be used to help write test cases.
+	These are to be used in conjunction with helm install --dry-run mode.
+	The output from dry run will be sent to each of these functions that would generate either `release` name or slice
+	of `Component`struct that help to run tests and validate.
+*/
```

Comment:
I don't think we should add this comment separately. If required, we can add the comments to the functions itself.

Reply:
Ok.
Removed.